diff -Naur pandas-1.3.5-original/setup.cfg pandas-1.3.5/setup.cfg
--- pandas-1.3.5-original/setup.cfg	2021-12-12 05:20:48.000000000 -0500
+++ pandas-1.3.5/setup.cfg	2022-04-11 09:56:11.666164455 -0400
@@ -46,6 +46,11 @@
 pandas_plotting_backends =
     matplotlib = pandas:plotting._matplotlib
 
+[options.exclude_package_data]
+* =
+    *.c
+    *.h
+
 [options.extras_require]
 test =
     hypothesis>=3.58
